src/styles/nav.scss

```
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    padding: 23px 0;
+
+    text-transform: uppercase;
+
+    @include fontStyleNormal(14px);
+
+    color: #000;
+  }
+
+  &__link:hover {
+    color: $colorLightBlue;
+  }
+}
+
+.nav__link--is-active {
+  color: $colorLightBlue;
+
+  &::after {
+    content: "";
+    display: block;
+    position: absolute;
+    bottom: 0;
+    height: 4px;
+    border-radius: 8px;
+    width: 100%;
+    background: $colorLightBlue;
+  }
+}
```

```suggestion
  &__link {
    position: relative;
    display: inline-block;
    padding: 23px 0;

    text-transform: uppercase;

    @include fontStyleNormal(14px);

    color: #000;

    &:hover {
      color: $colorLightBlue;
    }
  }

  &--is-active {
    color: $colorLightBlue;

    &::after {
      content: "";
      display: block;
      position: absolute;
      bottom: 0;
      height: 4px;
      border-radius: 8px;
      width: 100%;
      background: $colorLightBlue;
    }
  }
}
```
Что можно улучшить: 
можно вложить `&__link:hover` в `&__link`
можно вложить `.nav__link--is-active` в  `&__link`